AutomationPractice/src/main/java/components/ProductListComponent.java

```diff
@@ -0,0 +1,45 @@
+package components;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import pages.BasePage;
+
+import java.util.List;
+
+public class ProductListComponent extends BasePage {
+
+    @FindBy(xpath = "//div[@class='right-block-content row']/div[1]/span[1]")
+    private List<WebElement> productPrice;
+
+    @FindBy(css = "a[title='Add to cart']")
+    private List<WebElement> addToCartBtnList;
+
+    @FindBy(css = "span.ajax_block_products_total")
+    private WebElement confirm;
+
+    public ProductListComponent(WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    public boolean isOpened() {
+        return false;
```

Comment:
Posledica toga sto extendujes BasePage je i to da imas nepotrebne override-ove (koje moras da pises da bi zadovoljio sintaksu te moras da pises false/null). Pokusaj da komponentu pozivas preko stranice, ali da sama komponenta extend-uje neku osnovnu, zajednicku komponentu a ne BasePage